unstructured/documents/form_utils.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+import copy
+
+from unstructured.documents.elements import FormKeyValuePair
+
+
+def _kvform_rehydrate_internal_elements(kv_pairs: list[dict]) -> list[FormKeyValuePair]:
```

Comment:
Could you add doc strings to this function explaining what it does and where it's used? Also, since it's an internal function and is only used in `element.py`, and reason not to put it in `elements.py` instead of splitting it into a new moduel?

Reply:
Moved back to elements.py